webcompat/views.py

```diff
@@ -365,35 +365,14 @@ def show_issue(number):
                            json_data=to_str(issue_data[0]))
 
 
-@app.route('/me')
-def me_redirect():
-    """Set a redirect to /activity/<username>, for logged in users."""
-    if not g.user:
-        abort(401)
-    get_user_info()
-    return redirect(url_for('show_user_page', username=session['username']))
-
-
 @app.route('/activity/<username>')
 def show_user_page(username):
-    """Set the route for user activity.
-
-    (this dupes some of the functionality of /me, but allows directly visiting
-    this endpoint via a bookmark)
-
-    If the user is not logged in, send back a 401.
-    Make sure we have username and avatar details from Github
-    If the username matches, render the template as expected.
-    If it doesn't match, abort with 403 until we support looking at
-    *other* users activity.
+    """Retired route. 410 Gone.
+    Decision made on June 2020. See
+    https://github.com/webcompat/webcompat.com/issues/3347
     """
-    if not g.user:
-        abort(401)
-    get_user_info()
-    if username == session['username']:
-        return render_template('user-activity.html', user=username)
-    else:
-        abort(403)
+    msg = app.config['SHOW_USER_PAGE']
```

Comment:
In `/rate_limit`

```
    msg = app.config['SHOW_RATE_LIMIT']
```

is calling a text message defined in https://github.com/webcompat/webcompat.com/blob/d1c3b122fac11a2c6cc838c67cc358a23d5df8a6/config/environment.py#L113-L117

but here `SHOW_USER_PAGE` is not defined at all. So that will fail.


What about renaming `SHOW_USER_PAGE` into `410_MESSAGE` in environment.py?

and replace the two routes with.

```python
@app.route('/activity/<username>')
@app.route('/rate_limit')
def show_410_gone(username):
    """Retired routes. 410 Gone.

    * Decision made on June 2020. See
    https://github.com/webcompat/webcompat.com/issues/3347
    * Decision made on March 2017. See
    https://github.com/webcompat/webcompat.com/issues/1437
    """
    msg = app.config['410_MESSAGE']
    return (msg, 410, {'content-type': 'text/plain; charset=utf-8'})
```